test/test-lang.c

```diff
@@ -6,7 +6,7 @@
 
 #define ASSERT_KIND(_token, expected_kind)                                       \
   if (_token->kind != expected_kind) {                                           \
-    FAIL("Expected kind %d, got %s", #expected_kind, _token->kind);              \
```

Comment:
I was using this to write out the string representation of the symbol for the enum value that gets passed in here, is it not supported in whatever version of GCC is on macOS?

Speaking of which, would it be useful to try and lock the compiler to a specific standard (like C99 or C11) so that I don't mistakenly use any GCC-specific extensions?

Reply:
> I was using this to write out the string representation of the symbol for the enum value that gets passed in here, is it not supported in whatever version of GCC is on macOS?

Oh, now I see what you're trying to do. That's done with `__stringify`. Let me update the PR

> Speaking of which, would it be useful to try and lock the compiler to a specific standard (like C99 or C11) so that I don't mistakenly use any GCC-specific extensions?

Potentially, but I think clang supports most, if not all, GCC extensions.